cypress/support/step_definition/login_management/firstLoginSteps.spec.js

```diff
@@ -0,0 +1,57 @@
+import { Given, defineStep as And, Then, When } from "@badeball/cypress-cucumber-preprocessor";
+import { getUserCredentials } from "../../custom_commands/login.helper";
+import { visitPage } from "../../custom_commands/input.utils";
+
+
+And("I will logout from admin account", () => {
+	cy.clickOnElement('[data-testid="initials"');
+	cy.clickOnElement('[data-testid="logout"');
+});
+
+When("I login with new student account {string}", (environment) => {
+	cy.visitPage(environment, "/login");
+	cy.tryClickOnElement('[data-testid="submit-cloud-site"]');
+	cy.writeToInput('input[data-testid="username-email"]', "john.student@mail.tld");
+	cy.writeToInput('input[data-testid="password-email"]', "Password1!");
+	cy.clickOnElement('[data-testid="submit-login-email"');
+});
+
+When("I login with new teacher account {string}", (environment) => {
+	cy.visitPage(environment, "/login");
+	cy.tryClickOnElement('[data-testid="submit-cloud-site"]');
+	cy.writeToInput('input[data-testid="username-email"]', "john.teacher@mail.tld");
+	cy.writeToInput('input[data-testid="password-email"]', "Password1!");
+	cy.clickOnElement('[data-testid="submit-login-email"');
+});
```

Comment:
Code duplication, can be merged into one step with additional parametrization of username and password. Username and password must be set in the *.feature file.